backend/experiment/serializers.py

```diff
@@ -66,7 +67,7 @@ def serialize_experiment(experiment_object: Experiment, participant: Participant
         'started_session_count': get_started_session_count(experiment_object, participant),
         'finished_session_count': get_finished_session_count(experiment_object, participant),
         'description': experiment_object.description,
-        'image': experiment_object.image.file.url if experiment_object.image else '',
+        'image': serialize_image(experiment_object.image) if experiment_object.image else '',
```

Comment:
As image is now an object, I would fallback to a `None` here and avoid converting to a string type.

```suggestion
        'image': serialize_image(experiment_object.image) if experiment_object.image else None,
```